google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -35,6 +33,7 @@
 import java.util.logging.LogRecord;
 import java.util.logging.Logger;
 import java.util.logging.SimpleFormatter;
+import com.google.cloud.logging.Logging.WriteOption;
```

Comment:
Thanks for this! I had been avoiding using the formatter because it touches a lot of the comments in LoggingHandler. I have applied it now - if you want I can revert the changes to comments.